src/search/services/query-builder.service.js

```diff
@@ -538,7 +547,7 @@ function LuceneQueryBuilder(LuceneQueryParser, QueryTreeValidator, QueryTreeTran
         if (fieldGroup.implicitField) {
           field = fieldGroup.implicitField;
         } else {
-          throw 'Field name is implicit and not defined elsewhere.';
+          // throw 'Field name is implicit and not defined elsewhere.';
```

Comment:
Should this still be commented out?

Reply:
Yes otherwise it throws an error.
I don't really understand why it's there...